deployment/debian-package-x64/pkg-src/changelog

```diff
@@ -1,3 +1,9 @@
+jellyfin (10.6.0-1) unstable; urgency=medium
```

Comment:
I'm not sure this is needed, cc @joshuaboniface

Reply:
It's all needed until #2656 is merged. But I agree, let's get *that* in which already includes the version bump.